tests/testthat/test-check-ids-match.R

```diff
@@ -168,6 +168,25 @@ test_that("check_ids_match bidirectional arg looks only in one direction", {
   expect_equal(res2$data[[1]], c(3, 4))
 })
 
+test_that("check_ids_match bidirectional arg returns relevant data", {
+  meta <- data.frame(individualID = 1:4)
+  manifest <- data.frame(individualID = c(1:2, 5))
+  res1 <- check_ids_match(
+    x = meta,
+    y = manifest,
+    idcol = "individualID",
+    bidirectional = TRUE
+  )
+  res2 <- check_ids_match(
+    x = meta,
+    y = manifest,
+    idcol = "individualID",
+    bidirectional = FALSE
+  )
+  expect_equal(c(res1$data[[1]], res1$data[[2]]), c(5, 3, 4))
+  expect_equal(res2$data[[1]], 5)
```

Comment:
This one doesn't test the new behavior that we've added, as it would pass even if both sets of values were being returned. I'd propose something like:
```suggestion
  expect_equal(res2$data, list(`Missing from x` = 5))
  expect_equal(length(res2$data), 1)
```

Reply:
Oh! I see. That makes sense. Whoops. We need to check that it's not returning both. Thank you!